docs/GettingStarted.md

```diff
@@ -65,16 +65,19 @@ const selector = ({ color = 'red' } = {}) => ({
 })
 
 // OR operator
-const selector = props => ({
-  color: props.color || 'red'
-})
+const selector = props => {
+  props = props || {}
```

Comment:
This is not necessarily required as Fela uses an empty object as default, but to be sure it does not break one should use it, I guess :P